### DISCUSSION TITLE
migrate from portable to persistent installation

### DISCUSSION THREAD

**Discussion by maurerr** [ferdium/ferdium-app#1781] [General]
hello,

I have a portable installation on win10 x64.
Could you please help me with the steps to migrate to a full (persistent) installation without a need to reconfigure the services, if possible?

thanks !

**Comment by Alphrag** [comment]
Hey @maurerr. If you haven't done it so far here are a few points to understand what would happen. 
When running the Portable version, the app creates a local folder called `FerdiumAppData` in the same it is located at, which contains all the information for the app to run in terms of configuration of the services, the settings, whether is uses an account on a server or it is used without an account, the custom icons that have been saved, etc. 
When Ferdium is installed directly onto the computer, a similar folder is created inside a system folder instead of where the executable is located. For example, if installed on a "per user" basis, it is located in `Username\AppData\Roaming\Ferdium`. 

So, the best way to do a migration from a portable installation, would be to do the following steps:
1. locate the folder `FerdiumAppData` next to the portable installation;
2. install Ferdium from the autosetup executable;
3. launch the Ferdium just installed and in `Settings->Advanced` click on the "Open Profile Folder" button at the bottom of that panel, which will open the folder containing the files for that installation;
4. quit Ferdium completely;
5. replace the file in the folder opened in step 3., by the files from the folder of step 1.;
6. relaunch Ferdium, and it should now have the same configuration as the previous portable installation.

Let me know if this answers your question!

**Reply by maurerr** [reply]
thank you very much ! that worked very well !
